readme.md

```diff
@@ -12,8 +12,11 @@ Combine CSS files and Javascript files into single file in the correct order. Us
 
 ## Description
 
-TODO
+The cache is updated when the wp_enqueue version number changes or when the `Minit_Plugin:cache_bump()` method is fired.
 
+The complete cache can be purged by firing the `Minit_Plugin:cache_bump()` method.
```

Comment:
@ryanhellyer Let's suggest they use `do_action( 'minit-cache-version-bump' )` instead. That will avoid any issues if the plugin is de-activated or removed.